src/main/resources/migration/common/V15__add_resync_audit.sql

```diff
@@ -0,0 +1,11 @@
+create table if not exists resync_audit(
+    id bigserial NOT NULL CONSTRAINT pk_resync_audit PRIMARY KEY,
+    request jsonb not null,
+    requested_at timestamp not null,
+    requested_by varchar(32) not null,
+    requested_by_display_name varchar(255) not null,
+    source varchar(12) not null,
+    completed_at timestamp not null,
+    alerts_deleted uuid[] not null,
+    alerts_created uuid[] not null
+);
```

Comment:
- Worth adding an index on request prisonNumber to support the repo method?

Reply:
The resync was at the prisoner level. It makes sense to have prison number as a top level property here so I'd go with option 2.